OJP/OJP_Trips.xsd

```diff
@@ -112,6 +112,11 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="BaseTripPolicyGroup"/>
+			<xs:element name="BikeTransport" type="xs:boolean" default="false" minOccurs="0">
```

Comment:
Why is this one moved?

Reply:
I thought it would be better here. Don't you agree?

Reply:
I can agree on many things. I just notice _unrelated_ changes.

Reply:
Can change the title to reoptimising the optimisation part?

Reply:
Would be better.